python/README.md

```diff
@@ -70,3 +75,11 @@ print("Output: " + output)
 
 # Output: Protect humans, follow orders, survive.
 ```
+
+Hint: if you want to run this in a Jupyter notebook, you will need to
```

Comment:
Other way around. This is fine in the notebooks, but asyncio is needed for a separate test.py file or similar.

Reply:
Good catch, thanks!